rust/perspective-viewer/src/themes/monokai.less

```diff
@@ -64,7 +64,8 @@ perspective-string-column-style[theme="Monokai"] {
     --rt-neg-cell--color: #ff6188 !important;
     --rt--border-color: #444444;
 
-    // TODO: probably doesn't work with shadow DOM
+    // FIXME: broken in shadow DOM.  Can be fixed with a new custom property allowing empty header cell backgrounds
```

Comment:
I took a stab at this, but I couldn't work out how to declare the "default" value for the CSS custom property so that the theme override's was more specific than it